core-bundle/src/Resources/contao/classes/Frontend.php

```diff
@@ -291,84 +299,71 @@ public static function getRootIdFromUrl()
 	 */
 	public static function getRootPageFromUrl()
 	{
-		// HOOK: add custom logic
-		if (isset($GLOBALS['TL_HOOKS']['getRootPageFromUrl']) && \is_array($GLOBALS['TL_HOOKS']['getRootPageFromUrl']))
```

Comment:
Removing the hook seems like a BC break to me.

Reply:
The hook is triggered by the the new route provider (https://github.com/contao/contao/pull/95/files#diff-8f3e1abe0064e8534f4e9193ef359276R470)